apps/api/src/app/events/e2e/bridge-sync.e2e.ts

```diff
@@ -118,6 +110,42 @@ describe('Bridge Sync - /bridge/sync (POST)', async () => {
     expect(workflowData.steps[0].template?.name).to.equal('send-email');
   });
 
+  it('should create a workflow identified by a space-separated identifier', async () => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Thanks for adding this test!‏